src/commands/synthetics/run-tests-command.ts

```diff
@@ -254,16 +255,39 @@ export class RunTestsCommand extends Command {
         interval: toNumber(process.env.DATADOG_SYNTHETICS_OVERRIDE_RETRY_INTERVAL),
       })
     )
+    const envOverrideBasicAuth = deepExtend(
+      this.config.defaultTestOverrides?.basicAuth ?? {},
+      removeUndefinedValues({
+        password: process.env.DATADOG_SYNTHETICS_OVERRIDE_BASIC_AUTH_PASSWORD,
+        username: process.env.DATADOG_SYNTHETICS_OVERRIDE_BASIC_AUTH_USERNAME,
+      })
+    )
+    const envOverrideCookies = deepExtend(
+      this.config.defaultTestOverrides?.cookies ?? {},
+      removeUndefinedValues({
+        append: toBoolean(process.env.DATADOG_SYNTHETICS_OVERRIDE_COOKIES_APPEND),
+        value: process.env.DATADOG_SYNTHETICS_OVERRIDE_COOKIES_VALUE,
```

Comment:
What do you think of removing `DATADOG_SYNTHETICS_OVERRIDE_COOKIES_VALUE` in favor of `DATADOG_SYNTHETICS_OVERRIDE_COOKIES`?

The need for `.value` is just because we needed to add a `.append`, and I think it's redundant.

In my opinion, having:
- `DATADOG_SYNTHETICS_OVERRIDE_COOKIES='name1=value1;name2=value2'`
- `DATADOG_SYNTHETICS_OVERRIDE_COOKIES_APPEND=true`

and:
- `--override cookies='name1=value1;name2=value2'
- `--override cookies.append=true`

have the benefit of removing the possible exception `Cannot have both a string and an object for cookies`.